images/kindnetd/cmd/kindnetd/cni.go

```diff
@@ -28,27 +29,49 @@ import (
 	"k8s.io/utils/net"
 )
 
+//sensibleMTU value to fall back on - if we're not able to configure from eth0
+const defaultMtu = 1480
```

Comment:
why do you think we should change the default?

Reply:
We could allow the kernel to set the default, as documented here: https://github.com/containernetworking/plugins/tree/master/plugins/main/ptp#example-network-configuration 


I've typically found that mtu is configured to 1500, which has been problematic for many people.

Maybe it should be 1400 instead.

Reply:
I prefer to keep current behaviour, and allow the kernel to set it